hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -150,10 +150,13 @@ def get_all_transactions(self) -> Iterator['BaseTransaction']:
             assert tx is not None
             yield tx
 
-    def get_count_tx_blocks(self) -> int:
-        keys_bcount = self._db.get_property(b'rocksdb.estimate-num-keys', self._cf_tx)
-        keys_count = int(keys_bcount)
-        return keys_count
+    def get_vertices_count(self) -> int:
+        try:
+            return super().get_vertices_count()
+        except NotImplementedError:
```

Comment:
Why do we need the fallback for each storage type? We might rely on the index only and raise an exception if the index is not there. I feel we will make the index non-optional soon anyway.

Reply:
I talked about this in https://github.com/HathorNetwork/hathor-core/pull/496#discussion_r1003423365

Basically one test failed when I did this.

Reply:
It was refactored. Good job. Thanks!